src/main.cpp

```diff
@@ -296,10 +297,39 @@ std::vector<std::string> extra_flags(
   return extra_flags;
 }
 
-void CreateDynamicPasses(std::function<void(ast::Pass&& pass)> add)
+struct Args {
+  std::string pid_str;
```

Comment:
Thinking out loud: I guess one downside to this specific proposal is that we did lose the unification with existing args (e.g. `opts.pid` as a thing). But honestly I think that with the pid filtering working as is, it would be worth thinking about that some more and seeing if it still makes sense.

Reply:
Why couldn't we make those available (in the future) to `getopt` e.g. `getopt("pid")` ? Also what other current long options do you think users need to access in their programs?